README.md

```diff
@@ -1,70 +1,136 @@
 # Project _pod_ tato Head
```

Comment:
I think it would be wise to introduce a build-dependencies section.
For example, the `Makefile` will not run without `cosign` binary.
We also should make light of the `.env` required if someone wants to build this locally.

Reply:
Thanks @AlexsJones! I think you're suggesting that we introduce a build dependencies section and include the likes of `cosign`, `jq` and other dependencies there. Is that right? Makes sense to me, I'll add that to the main README.

**But** :wink:, I'd like to note that the script itself checks for `cosign` (and `trivy`) and doesn't even try to sign (or scan) images if it's not available. Also if you run `make install-dependencies` you'll get it - that's how we ensure it's available for the release task.